src/utils/room.ts

```diff
@@ -36,3 +45,47 @@ export const logIfCannotFindExistingParticipant = (
     );
   }
 };
+
+export interface VenueSpace {
+  text: string;
+  template?: VenueTemplate;
+  icon: string;
+}
+
+export const venueSpacesList: VenueSpace[] = [
```

Comment:
Shouldn't this be moved to a file inside `src/settings/` and re-exported from `src/settings/index.ts` additionally with ALL_CAPS_NAMING like the other enumerating lists?

Reply:
fixed in dc3dcc0